### PR TITLE
Added "Firmware Installation" tutorial.

### PR DESCRIPTION
**Pull request by dweston**
If this is of use to everyone I'll have a go at embellishing this with a version for Ethernet and also for set-up on macOS.

### CODE REVIEW

**Comment by MakerDockio** [comment]
Reads really well and thanks for your efforts. I'll let another person review before merging @sumnerboy12 you happy with this? 

I'm also happy for you @dweston to take a crack at macos / ethernet. 
In my opinion it would be nice if we could unify the installation steps to cover all scenarios so not bloat the instructions with multiple versions with slight differences, maybe highlighting any differences with platform / method i.e. wifi vs ethernet in each step in brackets or something.

**Comment by dweston** [comment]
Take your point on bloat.

I’m reasonably technically aware but still find I struggle with the finer points. Minor issues took me significant time to resolve (AKA that the web server code for the WiFi AP set-up would not render properly on a FireFox browser - it requires Chrome).

Hence I opted to really spell out the detail to ease the way for others who may follow.

File: docs/docs/firmware/touch-panel-esp32.md
Comment (sumnerboy12) [inline docs/docs/firmware/touch-panel-esp32.md:2409]: Could we match the case of the release binary files generated - i.e. `OXRS-IO-TouchPanel-ESP32-FW.*`?

**Review by sumnerboy12 — APPROVED** [review]
One very minor suggestion, but otherwise it looks great. Thanks!

**Comment by dweston** [comment]
Sure - I’ll fix shortly and push.

Took a bit of effort to set up dev environment but seems to be playing ball now. Am using VSCode which seemed to cough a bit gaining authorisation from GitHub.

**Pull request merged**
